src/pages/registration/resource_type_tab/sub_type_forms/exhibition_types/ExhibitionProductionForm.tsx

```diff
@@ -0,0 +1,142 @@
+import { OutputRow } from '../artistic_types/OutputRow';
+
+const exhibitionSubtypes = Object.values(ExhibitionProductionSubtype);
+type ExhibitionProductioModalType = '' | 'ExhibitionBasic';
+
+export const ExhibitionProductionForm = () => {
+  const { t } = useTranslation();
+  const { values, errors, touched } = useFormikContext<ExhibitionRegistration>();
+  const { subtype, manifestations } = values.entityDescription.reference.publicationInstance;
+
+  const [openModal, setOpenModal] = useState<ExhibitionProductioModalType>('');
+
+  return (
+    <>
+      <Field name={ResourceFieldNames.PublicationInstanceSubtypeType}>
+        {({ field, meta: { error, touched } }: FieldProps<ExhibitionProductionSubtype | ''>) => (
+          <StyledSelectWrapper>
+            <TextField
+              data-testid={dataTestId.registrationWizard.resourceType.subtypeField}
+              select
+              variant="filled"
+              fullWidth
+              {...field}
+              label={t('registration.resource_type.type_work')}
+              required
+              error={!!error && touched}
+              helperText={<ErrorMessage name={field.name} />}>
+              {exhibitionSubtypes.map((exhibitionType) => (
+                <MenuItem value={exhibitionType} key={exhibitionType}>
+                  {t(`registration.resource_type.exhibition_production.subtype.${exhibitionType}`)}
+                </MenuItem>
+              ))}
+            </TextField>
+          </StyledSelectWrapper>
+        )}
+      </Field>
+
+      {subtype?.type === ExhibitionProductionSubtype.Other && (
+        <Field name={ResourceFieldNames.PublicationInstanceSubtypeDescription}>
+          {({ field, meta: { error, touched } }: FieldProps<string>) => (
+            <TextField
+              data-testid={dataTestId.registrationWizard.resourceType.subtypeDescriptionField}
+              variant="filled"
+              fullWidth
+              {...field}
+              required
+              multiline
+              label={t('registration.resource_type.type_work_specified')}
+              error={!!error && touched}
+              helperText={<ErrorMessage name={field.name} />}
+            />
+          )}
+        </Field>
+      )}
+
+      <div>
+        <Typography variant="h3" component="h2" gutterBottom>
+          {t('registration.resource_type.artistic.announcements')}
+        </Typography>
+        <FieldArray name={ResourceFieldNames.PublicationInstanceManifestations}>
+          {({ push, replace, remove, move, name }: FieldArrayRenderProps) => (
+            <>
+              {manifestations.length > 0 && (
+                <Table sx={{ '& th,td': { borderBottom: 1 } }}>
+                  <TableHead>
+                    <TableRow sx={{ '& th,td': { borderBottom: 1 } }}>
+                      <TableCell>{t('common.type')}</TableCell>
+                      <TableCell>{t('registration.resource_type.artistic.exhibition_place')}</TableCell>
```

Comment:
noen i18n-nøkler som med fordel kunne vært flyttet ut av `artistic`, men gjør ikke dette nå
```suggestion
                      <TableCell>{t('registration.resource_type.exhibition_place')}</TableCell>
```